kubernetes/zenko/charts/mgob/templates/statefulset.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1 # Change to "apps/v1beta1" for Kubernetes < 1.10.
+kind: StatefulSet
+metadata:
+  name: {{ template "mgob.fullname" . }}
+spec:
+  serviceName: {{ template "mgob.fullname" . }}
+  replicas: 1
+  selector:
+    matchLabels:
+      role: {{ .Values.selector.role }}
+  template:
```

Comment:
Should have an `annotations` section with the checksum of the ConfigMap. This will force an update to the pods if the ConfigMap ever changes.
https://github.com/helm/charts/blob/master/stable/redis-ha/templates/redis-ha-statefulset.yaml#L20

Reply:
I was really missing this. Thank you, done  :)

Reply:
Great catch, I was missing this.